vagrant/install_agent.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
```

Comment:
Is this our latest version from https://github.com/Puppet-Finland/scripts/tree/master/bootstrap/linux? I'd hate to reintroduce bugs that have been fixed :smile:.